### DISCUSSION TITLE
User log UI missing entries

### DISCUSSION THREAD

**Discussion by putorti** [nyaruka/floweditor#1135] [Ideas]
One example we found, we can't grab the hook log with the link out of the user UI, the row where it would be is missing. Session explainer has it however.

<img width="1117" alt="image" src="https://user-images.githubusercontent.com/82857/73279835-acae1100-41bb-11ea-9640-478082959589.png">

![image](https://user-images.githubusercontent.com/82857/73279852-b0da2e80-41bb-11ea-9bd3-2b8e219e8221.png)

**Comment by rowanseymour** [comment]
Does the `WebHookResult` still exist in the database? Did the task to trim those get to it first?

**Comment by rowanseymour** [comment]
(this is definitely not a floweditor issue)
